lib/active_model/serializer/logging.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveModel
+  class Serializer
+    module Logging
```

Comment:
@bf4 are you saying everything should be switched to the new namespace before adding anything?

wouldn't that only matter depending on the order of merges? waiting on stuff like that causes needless delays, imo

but maybe I misunderstand? idk